src/Tools/dotnet-monitor/CollectionRules/ActionListExecutor.cs

```diff
@@ -83,7 +84,11 @@ public ActionListExecutor(ILogger<CollectionRuleService> logger, ICollectionRule
 
                         if (!_actionOperations.TryCreateFactory(actionOption.Type, out factory))
                         {
-                            throw new InvalidOperationException(Strings.ErrorMessage_CouldNotMapToAction);
+                            throw new InvalidOperationException(string.Format(
+                                CultureInfo.InvariantCulture,
```

Comment:
NOTE: We're pretty inconsistent with what culture we use when formatting error messages (some places use `CultureInfo.CurrentCulture`, while others use the invariant culture). This should be a separate effort to standardize everything but for now I've opted to just use the invariant culture here.